modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/RowVersion.java

```diff
@@ -112,7 +124,7 @@ boolean isTombstone() {
     }
 
     static boolean isTombstone(int valueSize) {
-        return valueSize == 0;
+        return valueSize <= 0;
```

Comment:
How is it possible that the size is negative? It's not mentioned in documentation for this class, or any other as far as I see

Reply:
If you look at the `org.apache.ignite.internal.storage.pagememory.mv.RowVersion#RowVersion(int, long, org.apache.ignite.internal.hlc.HybridTimestamp, long, java.nio.ByteBuffer)`, then if the `ByteBuffer` is `null`, then `valueSize` is equal to `-1`, I will fix it with 0.